docs_src/microservices/device/supported/device-usb-camera/walkthrough/deployment.md

```diff
@@ -9,31 +9,47 @@ Follow this guide to deploy and run the service.
         cd ~/edgex/edgex-compose/compose-builder
         ```
 
-    2. Run EdgeX with the microservice:  
+    2. Checkout the latest release ({{version}}):
 
-        - For non secure mode
-            ```
-            make gen ds-usb-camera no-secty
-            ```
-        - For secure mode 
-            ```
-            make gen ds-usb-camera
-            ```
-        - Docker Compose start command
-            ```
-            docker compose -p edgex up -d
-            ```
+        ```shell
+        git checkout {{version}}
+        ```
+    
+    3. Run EdgeX with the USB microservice in secure or non-secure mode:  
 
-        - Docker Compose clean command
-            ```bash
-            make clean
-            ```  
+        ##### Non-secure mode
+
+        ```shell
+        make run ds-usb-camera no-secty
+        ```
+
+        ##### Secure mode 
+
+        !!! note
+            Recommended for secure and production level deployments. 
+
+        ```shell
+        make run ds-usb-camera
+        ```
 
 === "Native"
-    1. Build the executable  
-    ```shell
-    make build
-    ```
+   
+    1. Navigate to the Edgex compose directory.
```

Comment:
Shouldn't they navigate to the device-usb-camera directory ?

Reply:
No. This should be change to "Navigate to edgex-compose/compose-builder directory